projects/knora-ui/src/lib/viewer/values/time-value/time-value.component.html

```diff
@@ -1,22 +1,29 @@
-<span [formGroup]="form" class="parent-component-wrapper">
-    <mat-form-field class="large-field child-value-component" floatLabel="never">
-      <kui-time-input #timeInput [formControlName]="'timeValue'" class="value" [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-time-input>
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-      <textarea matInput
-                cdkTextareaAutosize
-                #autosize="cdkTextareaAutosize"
-                cdkAutosizeMinRows="1"
-                cdkAutosizeMaxRows="6"
-                [formControlName]="'comment'"
-                class="comment"
-                placeholder="Comment"
-                type="text"
-                [readonly]="mode === 'read'"
-                spellcheck="false">
-        </textarea>
-    </mat-form-field>
-    <mat-error class="custom-error-message" *ngIf="form.controls.timeValue.hasError('valueNotChanged')">
-      Please change the value
-    </mat-error>
-</span>
\ No newline at end of file
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+  <p class="rm-value date">Date: {{valueFormControl.value | date}}</p>
```

Comment:
What happens if the value is `null`?

Reply:
fixed in 37865b7
now gives a console error and returns an empty string. Should the user be provided with some text?

Reply:
I meant the Angular date pipe: what happens if `valueFormControl.value` were `null`?

Reply:
ah, from my testing, nothing will be displayed if an invalid value is passed in